### DISCUSSION TITLE
【建议】纠正standalone模式中的部署文档中一处错误

### DISCUSSION THREAD

**Discussion by william-peng-cn** [apache/dolphinscheduler#13830] [General]
https://github.com/apache/dolphinscheduler/blob/3.1.4-release/docs/docs/zh/guide/howto/datasource-setting.md

![image](https://user-images.githubusercontent.com/48680780/228778471-297aef65-115c-41cc-ac12-3266283f30f9.png)

> 将mysql-connector-java驱动加到./standalone-server/libs/standalone-server/目录下, 

应当更正为

> 将mysql-connector-java驱动加到./tools/libs/目录下,

**Comment by muzijiang** [comment]
我安装文档步骤修改了元数据库，但是启动的时候仍然在找 org.h2.Driver 驱动，是否还有其他位置需要修改？

[INFO] 2023-04-19 13:25:23.442 +0000 com.zaxxer.hikari.HikariDataSource:[110] - HikariPool-1 - Starting...
[ERROR] 2023-04-19 13:25:23.447 +0000 org.apache.ibatis.mapping.VendorDatabaseIdProvider:[52] - Could not get a databaseId from dataSource
java.lang.RuntimeException: Driver org.h2.Driver claims to not accept jdbcUrl, jdbc:mysql://192.168.56.107:3306/dolphinscheduler?useUnicode=true&characterEncoding=UTF-8&useSSL=false&serverTimezone=GMT&allowPublicKeyRetrieval=true
        at com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:110)
        at com.zaxxer.hikari.pool.PoolBase.initializeDataSource(PoolBase.java:331)

**Reply by bberzhou** [reply]
我是启动的standalone-server，参考官方文档报错信息跟你类似，修改了apache-dolphinscheduler-3.1.5-bin/standalone-server/conf中的application.yaml配置文件里面datasource，重新启动成功。


![修改配置文件](https://user-images.githubusercontent.com/24271582/234802815-5f9324db-1b5b-4f4b-a3e1-7c102b6dc57c.jpg)

**Reply by fuchanghai** [reply]
cc @davidzollo @ruanwenjun @EricGao888
